Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListControlObjectTest.cs

```diff
@@ -631,7 +631,10 @@ public void TestGetRowMatchesExactly ()
       // Set Timeout to Zero so we don't have to wait the full timeout for the exception
       bocList.Scope.ElementFinder.Options.Timeout = TimeSpan.Zero;
 
-      Assert.That (() => bocList.GetRowWhere ("Title", "EO"), Throws.Exception.InstanceOf<MissingHtmlException>());
+      Assert.That (
+          () => bocList.GetRowWhere ("Title", "EO"),
+          Throws.Exception.InstanceOf<WebTestException>()

```

Comment:
Is that one the only WebException with a message that combines technical information (i.e. the CSS path) with specified information (i.e. the content value to look for)?